### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by zhao181** [dougsouza/pytorch-sync-batchnorm-example#3]
your explaination is very helpful. But I still could not implement it. Could you please give an example? Thanks.

**Comment by dougsouza** [comment]
Here's a complete example: https://github.com/pytorch/examples/blob/master/imagenet/main.py

Let me know if you have specific questions
